### DISCUSSION TITLE
[Q] LTS and who is using our roadrunner (in production)

### DISCUSSION THREAD

**Discussion by icy** [roadrunner-server/roadrunner#851] [Q&A]
Hi team,

First of all I'd like to say "woh" and thank you for the excellent idea `Roadrunner`. I've seen `php-fpm` deployments for years and really looking for something new :)

We're gonna give a try in our test environment. It may be great or not. And we would like to know our roadmap and long-term support (LTS). Our team worries if they will have to get back to `php-fpm` someday; so any hint would be helpful for our decision making process.

And who is using our tool in production? Is there any issue we need to know beforehand?

Thanks and  I do wish our product all the best.

**Comment by icy** [comment]
Thanks @rustatian for your moving the topic to this discussion. I didn't notice we had discussion boards. 

PS: I've fixed some typo errors.

**Comment by rustatian** [comment]
Hey @icy, nice to meet you. First of all, your can be sure, that RR will be supported long time and we won't leave the development)

> And who is using our tool in production? Is there any issue we need to know beforehand?

I'm not a PHP dev, but I'm sure that @wolfy-j will answer better about RR caveats for the PHP dev :)

**Comment by rustatian** [comment]
@icy Sorry for the late response, I was on sick leave (covid). 
> And who is using our tool in production?

Small and big (> 1000 empl) companies, users, sorry, I can't say names (NDA) :)

> Is there any issue we need to know beforehand?

I guess only read docs and start experimenting )) Also, we are always open to the feature requests, so, if your or your company uses some third party product which you wanted to be integrated as `http` middleware or JOBS driver or even a standalone plugin, make a feature request and we will help you. You can always ask a question in our discord server (en/ru): https://discord.gg/TFeEmCs

> Our team worries if they will have to get back to php-fpm someday;

RR is stable, `v2` newer got any (except configuration, but it also has versions and we support auto transform between configuration versions) breaking changes in the RR2 <-> PHP protocols (JOBS, Goridge, HTTP, TCP etc).
RR1 known since 2018, so, is a fairly stable, time-tested product.
